pkg/apis/serving/v1/roundtrip_test.go

```diff
@@ -19,20 +19,82 @@ package v1
 import (
 	"testing"
 
+	fuzz "github.com/google/gofuzz"
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/apitesting/fuzzer"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	pkgfuzzer "knative.dev/pkg/apis/testing/fuzzer"
 	"knative.dev/pkg/apis/testing/roundtrip"
 )
 
+// fuzzerFuncs includes fuzzing funcs for knative.dev/serving v1 types
+//
+// For other examples see
+// https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/fuzzer/fuzzer.go
+var fuzzerFuncs = fuzzer.MergeFuzzerFuncs(
```

Comment:
When we move it to a child `fuzzer` package we can drop the stutter

```suggestion
var Funcs = fuzzer.MergeFuzzerFuncs(
```

Reply:
Sure.

Reply:
Apparently doesn;'t work. v1->fuzzer->v1 dep loop